src/Facebook/FacebookBatchRequest.php

```diff
@@ -250,7 +257,10 @@ public function requestEntityToBatchArray(FacebookRequest $request, $requestName
             $batch['attached_files'] = $attachedFiles;
         }
 
-        // @TODO Add support for "omit_response_on_success"
+        if (isset($options['omit_response_on_success'])) {
```

Comment:
I think `name`, `attached_files` and current `options` can be merge in one `options` parameter.
And to resolve your issue with merging @SammyK: `$batch += $options;` ;)